src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -226,6 +226,15 @@ else if (FlutterSettings.getInstance().isDisablePreviewDart2()) {
     return new FlutterCommand(this, root.getRoot(), FlutterCommand.Type.RUN, args.toArray(new String[]{}));
   }
 
+  public FlutterCommand flutterRunOnTester(@NotNull PubRoot root, @NotNull String mainPath) {
+    final List<String> args = new ArrayList<>();
+    args.add("--machine");
+    args.add("--track-widget-creation");
```

Comment:
We're using the kernel transformer now?

Reply:
We don't use it yet, but might switch to it (and inspector) for widget / location mapping.
It will help with constant widgets.

Remove it for now maybe and reintroduce when we really going to use it?

Reply:
You're choice, I just wasn't sure it was in use here.

/cc @jacob314 for fyi

Reply:
Either option sounds fine to me.